scripts/start.js

```diff
@@ -0,0 +1,7 @@
+const childProcess = require("child_process");
+const fs = require("fs");
```

Comment:
Sorry, but why have you added this `import`?

Reply:
in this case, it's redundant, you're right

there was an option to copy files from script